tox.ini

```
@@ -34,8 +34,9 @@ commands =
 [testenv:mypy]
 deps =
     mypy>=1.0.0
+    pytest # for stubs
 commands =
-    mypy tenacity
+    mypy tenacity tests
```

```suggestion
    mypy
```
`files` has been modified so I guess it should work?

---

yepyep, fixed